supplementary_style_guide/glossary_terms_conventions/general_conventions/i.adoc

```diff
@@ -406,11 +406,12 @@ There is no functional difference between the first server that was installed an
 ==== image:images/yes.png[yes] ingress (noun, adjective)
 *Description*: In Red Hat OpenShift, _Ingress_  is a Kubernetes API object that developers can use to expose services within the cluster through an HTTP(S) load balancer and a proxy layer by using a public DNS entry. The `Ingress` resource defines the cluster-wide configuration for ingress traffic, and provides the ability to specify TLS options, a certificate, or a public CNAME that the OpenShift `IngressController` object can accept for HTTP(S) traffic. Additionally, _ingress_ can also be used to describe the incoming direction of network traffic. In Red Hat OpenShift, for example, this traffic is described as entering (ingress) or leaving (egress) an OpenShift cluster.
 
+Do not use _Ingress_ without markup, when referencing the `Ingress` resource or the `IngressController` object. Do not use _Ingress_ with markup when used to describe the direction of traffic.
```

Comment:
Removing the comma:
```suggestion
Do not use _Ingress_ without markup when referencing the `Ingress` resource or the `IngressController` object. Do not use _Ingress_ with markup when used to describe the direction of traffic.
```
Refactoring the double negative:
```suggestion
Use markup when referencing an `Ingress` resource or `IngressController` object. Do not mark up _ingress_ when describing the direction of traffic.
```

Reply:
@skrthomas What do you think of the adjustment the @rolfedh has here (the second one)?

I'd go with "Red Hat OpenShift" instead of just "OpenShift", but otherwise it LGTM. (Or maybe we need to also call out that it should be lowercase when it's the direction of traffic. wdyt?

Reply:
Oops, I am so sorry I somehow missed this in my GitHub inbox probably due to peer review squad influx 🙃 I think @rolfedh makes a great suggestion that clarifies this statement a lot.

Reply:
Thanks @skrthomas, updated!